packages/deployments/contracts/package.json

```diff
@@ -7,7 +7,7 @@
     "test:hardhat": "hardhat test",
     "test:forge": "yarn forge test --gas-report --hh",
     "forge": "forge",
-    "forge:install": "forge install",
+    "postinstall": "test -n \"$SKIP_FORGE\" || forge install",
```

Comment:
Can remove this postinstall script completely, it's already pushed to `amarok` branch.